### ISSUE TITLE
Incorrect appID

### ISSUE THREAD

**Issue by magic146** [apptentive/apptentive-ios#160]
```
- (NSString *)appID {
	NSString *appID = self.interaction.configuration[@"store_id"];
	if (!appID) {
		appID = [ATConnect sharedConnection].appID;
	}
	return appID;
}
```
If there is no "store_id" then this method doesn't get appID from [ATConnect sharedConnection] because appID is equal to empty string but not nil.

**Comment by magic146** [comment]
This is a fixed version:

- (NSString *)appID {
	NSString *appID = self.interaction.configuration[@"store_id"];
	if (appID.length == 0) {
		appID = [ATConnect sharedConnection].appID;
	}		
	return appID;
}

**Comment by pkamb** [comment]
Hello, thanks for reporting this. We are pushing out a fix for this on our backend that will not include the `store_id` key in the case of empty string values. This issue can also currently be fixed by adding a Store ID via your Apptentive dashboard.

We'll also make your suggested fix of checking for empty strings in the SDK (if you'd like to, please submit a PR).

Thanks!

**Comment by pkamb** [comment]
Hello, this is fixed in v2.0.3:

https://github.com/apptentive/apptentive-ios/releases/tag/v2.0.3
